packages/rocketchat-authorization/server/startup.js

```diff
@@ -34,6 +34,7 @@ Meteor.startup(function() {
 		{ _id: 'edit-room',                     roles : ['admin', 'owner', 'moderator'] },
 		{ _id: 'force-delete-message',          roles : ['admin', 'owner'] },
 		{ _id: 'join-without-join-code',        roles : ['admin', 'bot'] },
+		{ _id: 'leave-c',                       roles : ['admin', 'user', 'bot', 'anonymous'] },
```

Comment:
My only concern with doing this is for servers which have custom roles setup. I almost think we need to have a migration to ensure this gets added to all roles on the server except for `guest`. This way server administrators are forced to disable a person's ability to leave a channel manually instead of a new "odd behavior" happening.

Also, do you think we should create this for private groups as well? :thinking: